sdk/identity/identity/review/identity.api.md

```diff
@@ -77,10 +77,6 @@ export class AzureCliCredential implements TokenCredential {
 // @public
 export class AzurePowerShellCredential implements TokenCredential {
     constructor(options?: AzurePowerShellCredentialOptions);
-    protected getAzurePowerShellAccessToken(resource: string): Promise<{
```

Comment:
There's another one of these protected methods on AzureCliCredential.

Reply:
I'll make an issue to remove it for v2. That method was shipped as part of V1, sadly :/

Reply:
Issue: https://github.com/Azure/azure-sdk-for-js/issues/14970